services/API-service/src/api/lines-data/lines-data.service.ts

```diff
@@ -102,6 +102,7 @@ export class LinesDataService {
 
       const validationError = await validate(linesDto);
       if (validationError.length > 0) {
+        console.log('Seed line data validation error: ', validationError);
```

Comment:
Use `this.logger.log` instead of `console.log`. See [SeedAdminAreaData](https://github.com/rodekruis/IBF-system/blob/32a7ac8a02f0794112b4ec588258a525331c5c72/services/API-service/src/scripts/seed-admin-area-data.ts#L21) for reference.